src/weather_gen.cpp

```diff
@@ -17,6 +17,14 @@
 namespace
 {
 constexpr double tau = 2 * M_PI;
+constexpr double coldest_hour = 5;
+  // Out of 24 hours
+constexpr double daily_magnitude_K = 5;
+   // Greatest absolute change from a day's average temperature, in kelvins
+constexpr double seasonality_magnitude_K = 15;
+   // Greatest absolute change from the year's average temperature, in kelvins
+constexpr double noise_magnitude_K = 8;  // 2;
```

Comment:
`// 2` commented-out previous code?

Reply:
Yeah, that's the kind of code cleanup I'll do.